sample_shavar_list_creation.ini

```diff
@@ -20,8 +20,6 @@ remote_settings_password=%(SHAVAR_REMOTE_SETTINGS_PASSWORD)s
 [tracking-protection]
 # The location of the Disconnect list
 categories=Advertising|Analytics|Social
-# The location of the allowlist
-allowlist_url=https://raw.githubusercontent.com/mozilla-services/shavar-list-exceptions/master/allow_list
```

Comment:
Can we create an issue or link a PR to remove the `allowlist_url` from the .ini files in shavar-list-creation-config so that we don't forget to remove this. cc @englehardt, just wanted to let you know that we will be removing the allow_list as we discussed in our Shavar meeting.

Reply:
I see no reference to the allowlist in shavar-list-creation-config, so I guess we don't have to do anything.